src/cleed++/Core/Atom.hh

```
@@ -0,0 +1,64 @@
+#include <iostream>
+
+#include <Core/BaseElement.hh>
+
+using namespace std;
+
+namespace cleed {
+
+  class Atom : public BaseElement {
+    public:
+      Atom();
+      Atom(double x, double y, double z);
```

Clang (14.0.6)

[warning] parameter name 'x' is too short, expected at least 3 characters

(readability-identifier-length)